src/components/Sidebars/Items/BaseItem.js

```diff
@@ -49,27 +58,28 @@ const BaseItem = ({
           `}
         />
       )}
-      <div
+      <Link
+        external={false}
+        onClick={handleClickItem}
         css={`
-          display: flex;
-          padding: ${1.5 * GU}px;
-          ${!active &&
-            `&:hover {
-            background: rgb(246, 246, 247);
-          }`}
+          display: block;
+          width: 100%;
+          color: inherit;
```

Comment:
Swapped positions the Link and div so that in the case of the inner garden nav items the selection  takes the whole row

Reply:
Yeah, definitely better this way. I suppose I had a case of dyslexia here ^^!